esignet-mosipid.properties

```diff
@@ -191,21 +194,21 @@ mosip.esignet.supported-pkce-methods={'S256'}
 
 ## ---------------------------------------- Cache configuration --------------------------------------------------------
 
-mosip.esignet.cache.secure.individual-id=true
+mosip.esignet.cache.secure.individual-id=false
 mosip.esignet.cache.store.individual-id=true
 mosip.esignet.cache.security.secretkey.reference-id=TRANSACTION_CACHE
 mosip.esignet.cache.security.algorithm-name=AES/ECB/PKCS5Padding
 
 mosip.esignet.cache.names=clientdetails,preauth,authenticated,authcodegenerated,userinfo,linkcodegenerated,linked,linkedcode,linkedauth,consented,authtokens,bindingtransaction,vcissuance
 
-#spring.cache.type=redis
-#spring.cache.cache-names=${mosip.esignet.cache.names}
-#spring.redis.host=localhost
-#spring.redis.port=6379
+spring.cache.type=redis
+spring.cache.cache-names=${mosip.esignet.cache.names}
+spring.redis.host=redis-master-0.redis-headless.redis.svc.cluster.local
+spring.redis.port=6379
 management.health.redis.enabled=false
 
 # 'simple' cache type is only applicable only for Non-Production setup
-spring.cache.type=simple
+spring.cache.type=redis
```

Comment:
can we keep it simple and comment this property

Reply:
I believe IDA-plugin should need a transaction ID, hence the redis cache.